server_main/scripts/run.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright (c) 2015-2023 Contributors to the Eclipse Foundation
+# 
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+# 
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+
+SECRETS_DIR=/secrets
+DESTINATION=/tmp/userfiles/
+
+mkdir -p -m 2770 ${DESTINATION}
+
+cp /secrets/sslcerts/ca/ca_public_cert.pem ${DESTINATION}
+cp /secrets/sslcerts/device/tls.crt ${DESTINATION}
```

Comment:
Am I right in thinking that if these secrets change whilst we are running (e.g. cert is renewed) we won't notice.... or is this there something elsewhere watching the files?

Reply:
I'm thinking the readiness probe could check that if the secrets exist and are different to the ones in /var/lib/amlen/data/certificates then the probe could fail, which would result in the pod restarting which would then sort it all out. Does that seem reasonable?

Reply:
Yes - that seems reasonable (or something in the container can watch the file and restart the server process but either way - lets leave it as a separate item.